src/music.h

```diff
@@ -150,6 +151,11 @@ typedef struct
     /* Unload the library */
     void (*Unload)(void);
 
+    /* Start a specific track */
+    int (*StartTrack)(void *music, int track);
+
+    /* Get number of tracks. Returns -1 if not applicable */
+    int (*GetNumTracks)(void *music);
```

Comment:
Forgot to say, but I would put that under GetMetaTag() (or above) to keep the order of calls by role.